helm/chart/README.md

```diff
@@ -92,7 +92,7 @@ checks
 | `lifecycleOperator.manager.env.optionsControllerLogLevel`                     | sets the log level of Keptn Options Controller                         | `0`                                      |
 | `lifecycleOperator.manager.env.otelCollectorUrl`                              | Sets the URL for the open telemetry collector                          | `otel-collector:4317`                    |
 | `lifecycleOperator.manager.env.functionRunnerImage`                           | specify image for deno task runtime <!---x-release-please-version-->   | `ghcr.io/keptn/functions-runtime:v0.7.1` |
-| `lifecycleOperator.manager.env.pythonRunnerImage`                             | specify image for python task runtime <!---x-release-please-version--> | `ghcr.io/keptn/python-runtime:0.0.0`     |
+| `lifecycleOperator.manager.env.pythonRunnerImage`                             | specify image for python task runtime <!---x-release-please-version--> | `ghcr.io/keptn/python-runtime:v0.7.1`    |
```

Comment:
The python-runtime was not release yet, will be first released as part of 0.8.0. I guess we should not change this information

Reply:
hmm i just double checked. the release PR does bump the version correctly from `0.0.0` to `0.7.2`.
just the `v` is missing. i will change this back then and just add the `v`